specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-06-01/managedClusters.json

```diff
@@ -1015,6 +1015,202 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/privateEndpointConnections": {
```

Comment:
How many resources do you expect in this list? Do you need paging?

Reply:
This is actually the array in the property returned by a 'GET' request to NRP. So there is no paging.